account_move_import/wizard/import_move_line_wizard.py

```diff
@@ -86,11 +86,15 @@ def aml_import(self, cr, uid, ids, context=None):
 
         for ln in reader:
 
-            if not ln or ln and ln[0] and ln[0][0] in ['', '#']:
+            # skip blank lines in csv input
```

Comment:
@charbeljc 
The change you propose doesn't do the same thing. In the current version of this module lines can start with both an empty column or a '#' character before the header line. Changing this will most likely break some of the XLS input files used by the current install base of this module.

concerning the change on the csv file : please change the XLS sample input file and  generate the csv out of the xls in order to keep them in sync.

@gurneyalex 
ln = ln.strip() here makes sense (should be paired with ln[0].strip() -> ln[0] on line 98 infra).